scene/gui/rich_text_label.cpp

```diff
@@ -6173,6 +6334,20 @@ int RichTextLabel::get_total_glyph_count() const {
 	return tg;
 }
 
+int RichTextLabel::hit_test(const Point2 &coords) const {
+	ItemFrame *c_frame = nullptr;
+	Item *c_item = nullptr;
+	int c_line = 0;
+	int c_index = 0;
+	bool outside;
+
+	const_cast<RichTextLabel *>(this)->_find_click(main, coords, &c_frame, &c_line, &c_item, &c_index, &outside, false);
```

Comment:
Something is exceptionally off about this line but I can't quite put my finger on it.